draft-ietf-quic-recovery.md

```diff
@@ -366,21 +366,22 @@ below shows how the single timer is set based on the alarm mode.
 
 ### Handshake Packets
 
-The initial client hello has no prior RTT sample.  A client SHOULD remember
+The initial flight has no prior RTT sample.  A client SHOULD remember
 the previous RTT it observed when resumption is attempted and use that for an
 initial RTT value.  If no previous RTT is available, the initial RTT defaults
 to 200ms.  Once an RTT measurement is taken, it MUST replace initial_rtt.
 
-The client hello is retransmitted via a timer, similar to a tail loss probe, but
-it is expected client hellos will be acked immediately or a handshake packet will
-be sent immediately, so delayed acks do not need to be compensated for.  As such,
-it is always retransmitted at 2x the initial_rtt or smoothed_rtt.
+Endpoints MUST retransmit handshake packets if not acknowledged(1) within a
```

Comment:
Thanks, I removed it and tweaked the text about exponential backoff again.  Currently in the pseudocode(and the real code), all timers have their exponential backoff reset when an ack is received.  The idea being, if you're getting some packets through, based the retransmission time on RTT.  If you're not getting any data through, exponentially backoff because your RTT estimate may be wrong, and to avoid overloading the network.